src/devices/SensorHub/README.md

```diff
@@ -0,0 +1,10 @@
+﻿# SensorHub
+
+## Summary
+SensorHub for reading Temperature, Pressure, Illuminance and Humidity. I2C can be used to communicate with the device
+
+## Device Family
+
+**[Datasheet]**: [https://wiki.52pi.com/index.php/DockerPi_Sensor_Hub_Development_Board_SKU:_EP-0106]
+
```

Comment:
It is a good practice to add how to use the device in the Readme on top of the example.

Reply:
done